pokemongo_bot/cell_workers/pokemon_transfer_worker.py

```diff
@@ -75,7 +71,7 @@ def get_pokemon_potential(self, pokemon_data):
         for individual_stat in iv_stats:
             try:
                 total_iv += pokemon_data[individual_stat]
-            except Exception:
+            except:
```

Comment:
@surfaace just one last detail to be good to go for me: put back `except Exception` here.